### DISCUSSION TITLE
Resource saver map

### DISCUSSION THREAD

**Discussion by wipfli** [protomaps/basemaps#174] [Ideas]
I am thinking about making a "resource saver map" based on protomaps. The goal would be to make it optimized for bad internet connections and low-end devices.

Things that come to mind are:

- strong polygon and line simplification
- remove features from tiles
- simple layers
- precompute collisions of labels and turn them off in maplibre

So this would be a separate tileset and style with a similar look and feel as the normal basemap but just using less resources.

Would be cool if we had tools to measure and compare the performance of different tilesets and styles...

**Comment by bdon** [comment]
What if it was just the basemap with the POIs layer removed? More simplification may make the map look worse for little gain...

**Reply by wipfli** [reply]
That's a good idea. Also one could maybe push some details to higher zoom levels

**Comment by do-me** [comment]
That's a really cool idea! I'm also looking into something like the minimum viable basemap for a specific/different purposes. In the end, I guess that there is not "the one" map but that customization is crucial. 

Atm, there is no "simple" way of quickly generating different .pmtiles but I always need to change the java files manually right? Would be nice to have some kind of easy editor (like a GUI with checkboxes) or some kind of yaml file where to comment out what you don't need.
